models.py

```diff
@@ -26,22 +27,63 @@ def get_attr_by_path(self, path: str, sep: str = "__"):
 
         return current_level
 
-
     def get_indicators(self) -> dict:
         indicators = {}
         for indicator in self.indicators:
             value = self.get_attr_by_path(indicator["id"])
             cmp = indicator["not"] if isinstance(indicator["not"], list) else [indicator["not"]]
             indicators[f"has_{indicator['id']}"] = all(value != c for c in cmp)
+
         return indicators
 
+    def get_prefix_or_fallback_from(self, url: str, message: str) -> str:
+        m = re.match("^(https?://.*/)[^/]+$", url)
+
+        if m:
+            return m.group(1)
+
+        return message
+
+    def compute_prefix_id_source(self) -> str:
+        missing_prefix_message = 'Préfixe manquant'
+
+        try:
+            remote_id = self.payload['harvest']['remote_id']
+        except KeyError:
+            return missing_prefix_message
+
+        return self.get_prefix_or_fallback_from(
+            remote_id,
+            missing_prefix_message
+        )
+
+    def compute_prefix_url_source(self) -> str:
+        missing_prefix_message = 'Préfixe manquant'
+
+        try:
+            remote_id = self.payload['harvest']['remote_url']
+        except KeyError:
+            return missing_prefix_message
+
+        return self.get_prefix_or_fallback_from(
+            remote_id,
+            missing_prefix_message
+        )
+
+    def get_source_indicators(self) -> dict:
+        return {
+            'prefix_id_source': self.compute_prefix_id_source(),
```

Comment:
```suggestion
            'prefix_harvest_remote_id': self.compute_prefix_harvest_remote_id(),
```

Pour rester cohérent avec le nom data.gouv.fr.